src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/models/accrual/Agreement.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Agreement {
+
+  private UUID id;
+
+  private UUID tenantId;
+
+  @NotNull(message = "Person ID should not be null")
```

Comment:
We have code smells regarding unused private fields. I think we can rectify this by limiting the getter and setter to specific fields.

Currently we don't access id, tenantId